tests/integration/ha_tests/test_self_healing.py

```diff
@@ -155,19 +159,24 @@ async def test_kill_db_process(
     # Start an application that continuously writes data to the database.
     await start_continuous_writes(ops_test, app)
 
-    # Kill the database process.
-    await send_signal_to_process(ops_test, primary_name, process, "SIGKILL")
+    # Interrupt the database process.
+    await send_signal_to_process(ops_test, primary_name, process, signal)
+
+    # Wait some time to elect a new primary.
+    sleep(MEDIAN_ELECTION_TIME * 6)
```

Comment:
This and the other sleeps added allow for time to a new primary to be elected, much like we do on K8s